justfile

```
@@ -13,6 +13,9 @@ wait-for-ingress-controller:
 deploy-astria:
   kubectl apply -k kubernetes/
 
+wait-for-astria:
+  kubectl get -n astria-dev-cluster pods --no-headers=true | awk '/deployment/{print $1}' | xargs -I{} kubectl wait --namespace astria-dev-cluster --for=condition=ready pod/{} --timeout=600s
```

*technically* the question is not whether the deployment (or a pod within the deployment) is ready, but if geth (I think it's geth?) and the other parts are reachable. It's probably fine to wait for the deployment to be ready.

You could chain both the deployment being ready and ingress existing:
```suggestion
  kubectl wait -n astria-dev-cluster deployment astria-dev-cluster --for=condition=Available=True
```

(Could also test for `ingress` being ready, but this is probably fine; this also ignores updates, but that's probably fine as well.)

---

There is already a command for waiting for ingress that is performed before astria is even deployed though so it's not sufficient to just check for ingress. 

In my local testing, geth was actually ready before the deployment was ready, so waiting for the deployment was better than just waiting for geth.

---

There is a command for checking the ingress-*controller*, not for ingress itself (meaning its rules, etc).